src/ROUTES.js

```diff
@@ -96,4 +96,22 @@ export default {
             isParticipantsRoute: Boolean(lodashGet(pathSegments, 2)),
         };
     },
+
+    /**
+     * React Navigation is failing to parse urls with dot in the path segment.
+     * This method enable pasring of urls with dot in path Segment
+     * @param {String} path - Path for the config
+     * @returns {Object}
+     */
+    getEmailRouteLinkingConfig: path => ({
+        path,
+        parse: {
+            login: l => (l
+                ? decodeURIComponent(l).replace('@dot@', '.').replace(encodeURIComponent('@dot@'), '@dot@')
```

Comment:
What does the second `.replace()` achieve here? Didn't we replace with `.` already?

Reply:
I added this for safe parsing. In case the email already contains `@dot@` e.g. `rajat"@dot@"parashar@gmail.com`. Please let me know if I have forgotten to encounter any special cases.

`@dot@` I just took it based on the meaning. dot for `.` and `@` is a special URL character that allows us to safely convert emails that can contain `@dot@` as part of the string. 
`@` will be escaped and thus we can use it.

Reply:
>In case the email already contains @dot@ e.g. rajat"@dot@"parashar@gmail.com. Please let me know if I have forgotten to encounter any special cases.

Sorry, not following. Why would a valid email have more than one `@` let alone an `@dot@` in it?

Reply:
Ok. I saw somewhere that @ can be used in quotes. I believe I overthink it. I 'll remove the extra escaping.

Reply:
@marcaaron Updated the PR. Removed the Extra escaping stuff.